common/models.py

```
@@ -86,5 +87,11 @@ class VerificationMixin(models.Model):
     pan_card = models.ImageField(upload_to=pan_card_image_path, null=True, blank=True, storage=MediaStorage)
     pan_number = models.CharField(max_length=10, unique=True, editable=False)
 
+    def save(self, *args, **kwargs):
+        original = type(self).objects.get(pk=self.pk)
+        if original.aadhaar_number != self.aadhaar_number: raise ValidationError('Aadhaar Number once set cannot be changed!')
```

Flake8 (7.2.0)

[error] multiple statements on one line (colon)

(E701)